DigitalLearningSolutions.Web/Views/Shared/_Layout.cshtml

```diff
@@ -1,10 +1,22 @@
-﻿@using DigitalLearningSolutions.Web.Helpers
+﻿@using DigitalLearningSolutions.Data.Enums
+@using DigitalLearningSolutions.Web.Extensions
+@using DigitalLearningSolutions.Web.Helpers
+@using DigitalLearningSolutions.Web.Models.Enums
+@using DigitalLearningSolutions.Web.ViewComponents
+@using Microsoft.AspNetCore.Mvc.TagHelpers
 @using Microsoft.Extensions.Configuration
 @inject IConfiguration Configuration
 
 @{
   var headerExtension = (string)ViewData["Application"];
   var shouldDisplayHelpMenuItem = headerExtension != "Tracking System";
+  var selectedTab = (Tab)ViewData["SelectedTab"];
+  var applicationType = (ApplicationType)ViewData["ApplicationType"] == "" ? null : (ApplicationType)ViewData["ApplicationType"];

```

Comment:
Why are we doing a comparison to the empty string and then converting to null? Would we not expect it to either be null or populated?
Also use string.Empty